tests/test_views.py

```diff
@@ -60,7 +61,7 @@
     UserFactory, ApiOAuth2ApplicationFactory, ProjectFactory, WatchConfigFactory,
     NodeFactory, NodeLogFactory, AuthUserFactory, UnregUserFactory,
     RegistrationFactory, CommentFactory, PrivateLinkFactory, UnconfirmedUserFactory, DashboardFactory, FolderFactory,
-    ProjectWithAddonFactory, MockAddonNodeSettings,
+    ProjectWithAddonFactory, MockAddonNodeSettings, DraftRegistrationFactory
```

Comment:
In general try to keep imports sorted alphabetically; otherwise things can be hard to find (especially as this list continues to get longer). Applies (within the limits of PEP8) to most of this section.